src/app/components/table/table.component.scss

```
@@ -0,0 +1,225 @@
+	border-color: color-mix(in srgb, #ef4444 35%, transparent);
+	background: color-mix(in srgb, #ef4444 10%, transparent);
+	color: #ef4444;
+}
+
+.table-shell__error {
+	font-size: 0.875rem;
+	font-weight: 600;
+	color: #ef4444;
+}
+
+:host ::ng-deep .table-shell__table .p-datatable-table {
```

Stylelint (17.9.0)

[error] Unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)